### DISCUSSION TITLE
How to send logs from c# application?

### DISCUSSION THREAD

**Discussion by ZedZipDev** [datalust/seq-tickets#1433] [General]
How to send messages to Seq from a C# application (NET 4.5) directly without any Serilog, log4net etc. ?

**Comment by liammclennan** [comment]
You can use the HTTP API https://docs.datalust.co/docs/posting-raw-events

Note that the clients (Serilog etc) provide a lot of useful functionality.

**Comment by ZedZipDev** [comment]
Thhank you @liammclennan , You are right, Serilog is useful for applications. But I'd like to send logs to Seq from SQLCLR function and need more light way with minimal dependencies.
